app/assets/stylesheets/base/_sizes.scss

```diff
@@ -27,3 +27,4 @@ $size-46: 46px;
 $size-48: 48px;
 $size-50: 50px;
 $size-60: 60px;
+$size-73: 73px;
```

Comment:
Could you make this value 70px instead - just to be consistent with the even numbers we use?